### PR TITLE
Verify that long priority list can be used for smart variables (BZ1458817)

### PR DESCRIPTION
**Pull request by oshtaier**
```
nosetests tests/foreman/ui/test_classparameters.py -m test_positive_update_with_long_priority_list
.
----------------------------------------------------------------------
Ran 1 test in 160.010s

OK
```
```
nosetests tests/foreman/ui/test_variables.py -m test_positive_create_with_long_priority_list
.
----------------------------------------------------------------------
Ran 1 test in 95.172s

OK
```

### CODE REVIEW

File: tests/foreman/ui/test_classparameters.py [hunk 1]
```diff
@@ -1165,6 +1165,43 @@ def test_positive_create_matcher_attribute_priority(self):
...
+        :expectedresults: Smart class parameter is updated successfully and has
+            proper priority list
+
+        :BZ: 1458817
+
+        :CaseImportance: Medium
+        """
+        sc_param = self.sc_params_list.pop()
+        order_value = '\n'.join(
+            [gen_string('alpha').lower() for _ in range(40)])
```
Comment (ldjebran) [inline tests/foreman/ui/test_classparameters.py]: can we make an assert that value is more than 255 chars long, just to be to be sure, as gen_string without length,
also  order_value is of length  439 chars, can we use the same approximate length as the bug was verified with eg: 604 chars

**Comment by oshtaier** [comment]
```
nosetests tests/foreman/ui/test_classparameters.py -m test_positive_update_with_long_priority_list
.
----------------------------------------------------------------------
Ran 1 test in 154.536s

OK

nosetests tests/foreman/ui/test_variables.py -m test_positive_create_with_long_priority_list
.
----------------------------------------------------------------------
Ran 1 test in 133.842s

OK
```

File: tests/foreman/ui/test_classparameters.py [hunk 2]
```diff
@@ -1165,6 +1165,44 @@ def test_positive_create_matcher_attribute_priority(self):
...
+            proper priority list
+
+        :BZ: 1458817
+
+        :CaseImportance: Medium
+        """
+        sc_param = self.sc_params_list.pop()
+        order_value = '\n'.join(
+            [gen_string('alpha').lower() for _ in range(60)])
+        self.assertGreater(len(''.join(order_value)), 255)
```
Comment (ldjebran) [inline tests/foreman/ui/test_classparameters.py]: order_value is a string  we do not need to join

Comment (oshtaier) [reply tests/foreman/ui/test_classparameters.py]: oops, right, I was thinking about list

**Comment by oshtaier** [comment]
```
nosetests tests/foreman/ui/test_classparameters.py -m test_positive_update_with_long_priority_list
.
----------------------------------------------------------------------
Ran 1 test in 181.806s

OK
```

**Pull request merged**